pkg/loop/loopinstall/README.md

```diff
@@ -0,0 +1,75 @@
+# loopinstall
+
+A tool for installing Chainlink LOOP plugins from YAML configuration files.
+
+## Installation
+
+To install from source:
+
+```bash
+go install github.com/smartcontractkit/chainlink-common/pkg/loop/loopinstall@latest
+```
+
+Or clone and install locally:
+
+```bash
+git clone https://github.com/smartcontractkit/chainlink-common.git
+cd chainlink-common
+go install ./pkg/loop/loopinstall
+```
+
+## Usage
+
+```bash
+# Run `loopinstall --help` to see the options.
+loopinstall [options] <plugin-config-file> [<plugin-config-file>...]
+```
+
+## Configuration
+
+_(See `plugins.example.yaml` for a complete example configuration.)_
+
+Example configuration file structure:
+
+```yaml
+defaults:
+  goflags: "-ldflags='-s -w'" # Default Go build flags
```

Comment:
Why do we set these flags?
- `-s    disable symbol table`
- `-w    disable DWARF generation`

Reply:
Oh those were old docs. These seem important, but if we truly need `-s` then it looks like `-w` can be left off:
```
-s
	Omit the symbol table and debug information.
	Implies the -w flag, which can be negated with -w=0.
-w
	Omit the DWARF symbol table.
```